README.md

```diff
@@ -1,67 +1,141 @@
+> This example is designed to get you quickly writing and interacting with smart contracts on the NEAR blockchain using JavaScript! 🚀
 
-There are several other branches with different completed contracts that you can use as reference:
-- `coin-flip-skeleton` contains the skeleton code for the coin flipping game
-- `coin-flip-finished` contains the finished code for the coin flipping game
-- `coin-flip-hub` contains the finished code for the hub contract.
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/near-examples/near-sdk-js-quickstart)
 
-To test the fully working contract alongside the frontend, switch to the `coin-flip-finished` branch and follow the steps outlined in the [Quickstart](#quick-start) section.
+---
 
-### Prerequisites
+## Table of Contents
 
-In order to successfully complete this quickstart guide, you'll need to have a few things installed:
-- [Node.js](https://nodejs.org/en/about/) and [npm](https://www.npmjs.com/):
-```bash
-curl -sL https://deb.nodesource.com/setup_17.x | sudo -E bash -  
-sudo apt install build-essential nodejs
-PATH="$PATH"
-```
-Ensure that they are both installed by running a version check:
-```
-node -v
-npm -v
-```
+- [Requirements](#requirements)
+- [Quick Start](#quick-start)
+- [Exploring the Code](#exploring-the-code)
+  - [Smart Contract](#smart-contract)
+  - [Front end](#front-end)
+- [Example Game](#example-game)
+- [Help](#help)
+- [Feedback](#feedback)
 
-It's important to have the **newest** version of the [NEAR-CLI](https://docs.near.org/docs/tools/near-cli) installed such that you can make use of the JavaScript features. To install or update, run: 
+---
 
+## Requirements
+
+- [Node.js & npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+- [near-cli](https://docs.near.org/docs/tools/near-cli) `v3.1.1` or greater
+
+```bash
+npm i -g near-cli
 ```
-npm install -g near-cli
-```
+
+---
 
 ## Quick Start
-To run this project locally, install the dependencies and run the start script. This will build your contract to `build/contract.base64` which will then be deployed to a dev account. The frontend will then be built and started locally.
 
+1. Install dependencies
+
+```bash
+npm i
 ```
-npm i && npm run start
+
+2. Launch dApp! 🚀
+
+```bash
+npm run start
 ```
 
-To simply build the contract:
+> This script performs the following:
+>
+> - Compiles smart contract located in `./src/index.js` and exports to `./build/contract.base64`
+> - Creates a NEAR [`dev-account`](https://docs.near.org/docs/concepts/account#dev-accounts) with 200 Ⓝ `testnet` tokens. _(If it doesn't already exist)_
+> - Deploys the compiled smart contract to the `dev-account` on NEAR's [`testnet`](https://docs.near.org/docs/concepts/networks#testnet)
+> - Launches a local front-end that is connected to the smart contract (dApp)
+
+---
+
+## Exploring the code
+
+```bash
+.
+├── frontend
+│   ├── App.js
+│   ├── assets
```

Comment:
This hierarchy isn't the same as the one outlined in the frontend section

Reply:
Hmmmm... I don't see the diffrence.